docs/book/products.rst

```diff
@@ -4,69 +4,113 @@
 Products
 ========
 
-**Product** model represents unique products in your Sylius store. Every product can have different variations or attributes and has following values:
+**Product** model represents unique products in your Sylius store. Every product can have different variations or attributes.
 
-* *name* - The full name of the product
-* *slug* - Urlized version of the name
-* *description* - Description of the product
-* *shortDescription* - Simple description of the product for lists and banners
-* *metaDescription* - Description for search engines
-* *metaKeywords* - SEO keywords
-* *createdAt* - Date of creation
-* *updateAt* - Timestamp of the most recent update
+.. note::
+
+   Each product has to have at least one variant to be sold in the shop.
 
 Options
 -------
 
-In many cases, you will have product with different variations. The simplest example would be a T-Shirt available in different sizes and colors.
+In many cases, you will want to have product with different variations.
+The simplest example would be a piece of clothing, like a T-Shirt available in different sizes and colors
+or a glass available in different shapes or colors.
 In order to automatically generate appropriate variants, you need to define options.
 
 Every option type is represented by **ProductOption** and references multiple **ProductOptionValue** entities.
 
+For example you can have two options - Size and Color. Each of them will have their own values.
+
 * Size
     * S
     * M
     * L
     * XL
     * XXL
+
 * Color
     * Red
     * Green
     * Blue
 
+After defining possible options for a product let's move on to **Variants** which are in fact combinations of options.
+
 Variants
 --------
 
 **ProductVariant** represents a unique combination of product options and can have its own pricing configuration, inventory tracking etc.
 
 You are also able to use product variations system without the options at all.
```

Comment:
I would even say, that a variant concept should be described before an option part. Variants doesn't have to be based on options, but options are available via variants.